### ISSUE TITLE
cortex-m-rt v0.7.0 support

### ISSUE THREAD

**Issue by pdgilbert** [stm32-rs/stm32f3xx-hal#298]
(Related to #283)

I need `cortex-m-rt v0.7.0` to avoid conflicts with other crates. The github version `CHANGELOG.md` is reporting
>Support cortex-m-rt v0.7.0 but still allow v0.6.13 (#283)

but there seems to be a problem that `stm32f3` is specified in Cargo.toml as
```
stm32f3 = { version = "0.13.2", default-features = false }
```
I think `cortex-m-rt v0.7.0` support in `stm32f3` is only introduced in `0.14`. I get

```
error: failed to select a version for `cortex-m-rt`.
    ... required by package `stm32f3 v0.13.2`
    ... which is depended on by `stm32f3xx-hal v0.8.1 (https://github.com/stm32-rs/stm32f3xx-hal#91b47e12)`
    ... which is depended on by `rust-integration-testing-of-examples v0.1.0 (/home/paul/githubClones/rust-integration-testing/dev-testing)`
versions that meet the requirements `^0.6.10` are: 0.6.15

the package `cortex-m-rt` links to the native library `cortex-m-rt`, but it conflicts with a previous package which links to `cortex-m-rt` as well:
package `cortex-m-rt v0.7.0`
    ... which is depended on by `rust-integration-testing-of-examples v0.1.0 (/home/paul/githubClones/rust-integration-testing/dev-testing)`

failed to select a version for `cortex-m-rt` which could resolve this conflict
```

**Comment by Sh3Rm4n** [comment]
Ah finally I understand the problem. #283 is solving a problmen, which isn't to solve, because `stm32f3` expects a fixed versino `cortex-m-rt-0.7.0` for `v0.14.0` and `cortex-m-rt-0.6.x` for `v0.13.1`? If than, yes #282 should fix the problem.

**Comment by pdgilbert** [comment]
My guess is that is correct. (I'm still a newbie so it is just a guess.)
